plugin/neogit.vim

```diff
@@ -7,10 +7,18 @@ if !luaeval("require 'neogit.bootstrap'")
   finish
 endif
 
+function! neogit#refresh_manually(file)
+  call luaeval('(function() require "neogit.status".refresh_manually(_A) end)()', a:file)
+endfunction
+
 function! s:refresh(file)
   if match(bufname(), "^\\(Neogit.*\\|.git/COMMIT_EDITMSG\\)$") == 0
     return
   endif
+
+  if match("dirvish", &filetype) == 0
```

Comment:
I think this is no longer required right?

Reply:
Done :)